test/foundry/unit/concrete/erc4337account/TestERC4337Account_EntryPoint.t.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "../../../utils/Imports.sol";
+import "../../../utils/SmartAccountTestLab.t.sol";
+
+contract TestERC4337Account_EntryPoint is Test, SmartAccountTestLab {
+
+    function setUp() public {
+        init();
+    }
+
+    function test_CorrectEntryPointAddress() public {
+        assertEq(BOB_ACCOUNT.entryPoint(), address(ENTRYPOINT), "Should return the correct EntryPoint address");
```

Comment:
we also want to cover tests of being able to query implementation address, account version (if to be defined) etc
cc @filmakarov

Reply:
not sure to understand this, TestAccoungConfig_AccountId is already testing it.